test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -64,6 +64,9 @@ test
     await t.expect(pageSubtitle).contains('+1 XXX-XXX-2342');
     await t.expect(primaryButtonText).contains('Send a code via SMS');
     await t.expect(secondaryButtonText).contains('Receive a voice call instead');
+
+    await t.expect(await challengePhonePageObject.signoutLinkExists()).ok();
```

Comment:
Signout shouldn't exist on this page.

Reply:
why ?

Reply:
Yes, the screenshots don't have it, moreover, I believe user is only signed in `after` this step? Hence shouldn't show `sign out` here?

Reply:
it may not works like this. what if user want to switch to another username on the 2nd factor challenge page?